pytest-embedded-idf/pytest_embedded_idf/dut.py

```diff
@@ -1,9 +1,22 @@
+import io
+import os
+import sys
+
 from pytest_embedded import PexpectProcess
 from pytest_embedded_serial.dut import SerialDut
 
 from .app import IdfApp
 from .serial import IdfSerial
 
+try:
```

Comment:
I think a simple `import esptool` is safe here. The setup.py would install the esptool package.

Reply:
The `IdfDut` would only be used when `esp, idf` services are all enabled

```python
if 'esp' in _services and 'idf' in _services:
    from pytest_embedded_idf.dut import IdfDut

    classes[fixture] = IdfDut
```